### PR TITLE
Fb reward rate display

### PR DESCRIPTION
**Pull request by XDapps**
Added API call to Next.js back-end to retrieve reward rates.
Added columns to display reward rates on Pool details page and staking page
Extracted hooks from single file into individual files.
Fixed some broken links on the Staking Page in the pool statistic columns.

https://www.pivotaltracker.com/story/show/187605784
https://www.pivotaltracker.com/story/show/187610760
https://www.pivotaltracker.com/story/show/187610759
https://www.pivotaltracker.com/story/show/187605261
https://www.pivotaltracker.com/story/show/187605205

### CODE REVIEW

File: apps/web-staking/src/app/hooks/hooks.ts
Comment (Metroxe) [inline apps/web-staking/src/app/hooks/hooks.ts:1]: Really appreciate this!

**Review by Metroxe — APPROVED** [review]

File: apps/web-staking/.gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .yarn/install-state.gz
+/.vscode
```
Comment (bsCryptit) [inline apps/web-staking/.gitignore:8]: Why is this added here?

Comment (CryptITAustria) [reply apps/web-staking/.gitignore:8]: Needed to be added because it was not in the web-staking until now and we don't want to commit the .vscode yet

File: packages/core/src/data-centralization/types.ts [hunk 2]
```diff
@@ -26,7 +26,9 @@ export interface IPool {
     visibility: 'active' | 'inactive' | 'banned',
     network: string,
     updatedAt?: Date
-    createdAt: Date
+    createdAt: Date,
+    keyRewardRate: number,
+    esXaiRewardRate: number
```
Comment (CryptITAustria) [inline packages/core/src/data-centralization/types.ts:31]: This should already be done in this PR for the data sync story
https://github.com/xai-foundation/sentry/pull/193/

In the next step, we will remove the duplicate types and share one type for all projects in the monorepo, this is depending on finishing this story https://www.pivotaltracker.com/story/show/187709471

**Review by CryptITAustria — APPROVED** [review]

**Review by CryptITAustria — CHANGES_REQUESTED** [review]
@XDapps 

For testing we deployed this to the  [depricateddev instance](https://xai-staking-int.dev.cryptit.at/) 

These are the current issues:

- In the `Available Pools` switching to `Key Staking` does not switch to Key Reward, still shows esXAI Rate in %
image

- In `Staked pools` the tooltip is not working properly
image

- For both tables the column headers should be right aligned like in the designs
https://xd.adobe.com/view/abf38d19-1b01-49fd-b74b-dd12c5bdd237-fea0/screen/12b39442-f92b-4b60-bbec-7f8f77038278/

- It seems that the `Staked pools` list is not not loading one by one, but all at once, for more than 2 pools this might lead to a bad UX since my initial reaction would be to refresh the page becasue the table is not visibale at all. That would lead to the Stake pools list never showing up. We should still be able to render the pools one by one or request a design for a loading state

**Comment by XDapps** [comment]
Edit from Jerry: currently the 'Staked Pools' are being loaded all at once in the useGetUserInteractedPools hook. This was the existing design before this story. In order to load individually, it would require a refactor of the hook and the api call. Would you prefer that, or a loading state? We may need to update the story.
